src/Delete.js

```diff
@@ -0,0 +1,19 @@
+import './App.css'
+
+function Delete() {
+    const pageUrl = window.location.href
+    const id = pageUrl.substring(35,38);
```

Comment:
Not a very good way to access the id. What if the route changes? are you going to manually edit 35 & 38 ? What if we have interviews > 10,000? Your app is going to break. There is a hook for what you're trying to achieve.

Reply:
This worked:         const id = this.props.match.params.id;